source/index.rst

```diff
@@ -1,9 +1,16 @@
 地震“学”参考书
 ##############
 
-
 .. toctree::
    :maxdepth: 3
    :hidden:
 
    ray-theory-travel-times/index
+
+
+欢迎阅读由\ `地震“学”小组 <https://github.com/orgs/seismo-learn/people>`__\
+撰写的《地震“学”参考书》。
+
+.. warning::
+
+   本参考书正在缓慢编写中，尚未完成，敬请期待最终完成版。
```

Comment:
I'm afraid the book will never be finished.

Reply:
We may mean version 1.0.